stdlib/public/core/VarArgs.swift

```diff
@@ -75,11 +75,24 @@ internal let _x86_64CountGPRegisters = 6
 // from 8 to 16 based on reading the spec, probably the bug you're looking for
 // is elsewhere.
 @_versioned
-internal let _x86_64CountSSERegisters = 8
+internal let CountSSERegisters = 8
 @_versioned
-internal let _x86_64SSERegisterWords = 2
+internal let SSERegisterWords = 2
 @_versioned
-internal let _x86_64RegisterSaveWords = _x86_64CountGPRegisters + _x86_64CountSSERegisters * _x86_64SSERegisterWords
+internal let RegisterSaveWords = CountGPRegisters + CountSSERegisters * SSERegisterWords
+#elseif arch(s390x)
+@_versioned
+internal let CountGPRegisters = 16
+@_versioned
+internal let RegisterSaveWords = CountGPRegisters
+#endif
+
+#if arch(s390x)
+typealias VA_UInt = CUnsignedLongLong
+typealias VA_Int  = Int64
+#else
+typealias VA_UInt = CUnsignedInt
+typealias VA_Int  = Int32
```

Comment:
Similar issue here. These should be explicitly `internal typealias` and not just `typealias`, should be `CamelCase`, and should have a leading underscore because they are internal: `_VAInt`, etc.

That said, these shouldn’t be necessary: why aren’t you using `numericCast`?